api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
+
     # overrides ODMFilterMixin
     def get_default_odm_query(self):
         user = self.get_user()
         return (
-            Q('contributors', 'eq', user) &
+            Q('contributors', 'contains', user._id) &
+            Q('is_folder', 'ne', True) &
             Q('is_collection', 'ne', True) &
-            Q('is_deleted', 'ne', True)
+            Q('is_deleted', 'ne', True) &
+            Q('is_registration', 'ne', True)
```

Comment:
I was reconciling behavior currently on develop, and Caner's dashboard-redesign-component-3 that I am merging in with.  Currently, on develop, the NodeList endpoint /v2/nodes/ only returns nodes that are not deleted, are not folders, and are not registrations.  
![screen shot 2016-01-27 at 9 42 38 am](https://cloud.githubusercontent.com/assets/9755598/12616687/6477610a-c4da-11e5-80b6-c0ad6c427b7c.png)

On Caner's PR, there is instead a restriction on 'collection' instead of 'folder'.  There has been a lot of back and forth as to what should be included in the node list, but I think the users/me/nodes endpoint has been forgotten.  The users/me/nodes/ endpoint should be returning the same thing as v2/nodes/ except it is further restricted to just the user's projects.  When a user is looking through a list of projects, she shouldn't see deleted projects, or registrations of these projects, or collections(whatever this is), or folders.  So what I have here is doing exactly what /v2/nodes/ is doing except I've taken 'folders' out of the list too.  If folders are in the list, then the 'Dashboard' node comes up, which I certainly don't think is intended to show up in the user nodes list. 

**In summary** My additions to users/me/nodes/
- 'is_folder' excluded in addition to 'is_collection'. We had 'is_folder' before, and without this, the dashboard node shows up.
- 'is_registration' excluded.  It is excluded from /v2/nodes/ so it should also be excluded from /v2/users/me/nodes.  There are separate endpoints /v2/registrations/ and v2/users/me/registrations/.

I know there was a lot of back and forth as to whether registrations should be included in the endpoints, so I think things were changed without bringing all the docs/filtering/other endpoints up to date.  If registrations are excluded from /v2/nodes/ you shouldn't be able to filter on this so it should be removed from filterable fields and the docs.